app/views/events/show.html.erb

```diff
@@ -27,6 +27,12 @@
   <strong>Capacity:</strong>
   <%= @event.capacity %>
 </p>
+<p>
+  <strong>Demographics:</strong>
```

Comment:
this should only be shown if you're logged in as an admin.

Reply:
Since the graph is anonymous I think this is actually fine as it is.  Especially since the gender stat will most likely be removed once she talks it over with her team.